app-android/app/src/main/res/values-night-v31/styles.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<style name="SplashTheme" parent="Theme.AppCompat.Light.NoActionBar">
+		<item name="android:colorPrimary">@color/green</item>
+		<item name="android:colorPrimaryDark">@color/green</item>
+		<item name="android:windowBackground">@color/darkLighter</item>
+		<item name="android:statusBarColor">@color/white</item>
```

Comment:
this is `values-night`  but status bar is still white? is this intended?

Reply:
I changed it, but I think that doesn't make difference since we handle the app theme by ourselves on the webapp.
Maybe we should remove it in future releases?

Reply:
wouldn't it show up on the splash screen tho?

Reply:
At least on devices that I tested, the statusBarColor made no difference, the splash screen took the whole screen with the windowBackground color